Automattic/wp-calypso
# Stats: Release subscribers section to the public

## Proposed Changes

![image](https://github.com/Automattic/wp-calypso/assets/4044428/236800fd-2352-4844-8737-7d6d7a370975)

* enabling the new subscriber stats section to the public.
* adding a feature flag to hide the subscribers chart and overview section consuming a mock API.
* removing a feature flag from email stats in the subscriber stats page.
* removing a tooltip from the all-time section because we no longer compare visible values
* adding top padding to the grid section at the bottom of the page to separate it from the top sections
* extending the page with properties needed for the email component to properly redirect to its summary page

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug`
* Navigate to the subscriber stats section and verify that it loads without a line chart
* Add the query string `?flags=stats/subscribers-section,stats/subscribers-chart-section` and verify that it loads with a chart

## Pre-merge Checklist
